cgi-bin/DW/Controller/Editor.pm

```diff
@@ -0,0 +1,119 @@
+# DW::Controller::Editor
+#
+# Single-purpose routes for setting the editor userprops, which determine the
+# default formatting type when writing new entries/comments/etc in the web UI.
+#
+# Authors:
+#   Nick Fagerlund <nick.fagerlund@gmail.com>
+#
+# Copyright (c) 2020 by Dreamwidth Studios, LLC.
+#
+# This program is free software; you may redistribute it and/or modify it under
+# the same terms as Perl itself.  For a copy of the license, please reference
+# 'perldoc perlartistic' or 'perldoc perlgpl'.
+#
+package DW::Controller::Editor;
```

Comment:
nit: I know we're all over the map on this, but I think best practice now is to have a preamble like:

```perl
use v5.10;
use strict;
```